README.md

```diff
@@ -14,6 +14,23 @@ BotKube is participating in Hacktoberfest 2020. We are giving some really cool s
 ## Getting started
 Please follow [this](https://www.botkube.io/installation/) for complete BotKube installation guide.
 
+### Install BotKube to the Slack workspace
+
+Follow the steps below to install BotKube Slack app to your Slack workspace.
+
+Install BotKube Slack app to your Slack workspace
+Click the Add to Slack button provided to install BotKube Slack application to your workspace. Once you have authorized the application, you will be provided a BOT Access token. Kindly note down that token as it will be required while deploying BotKube backend to your Kubernetes cluster.
+
+[![slack_add](https://platform.slack-edge.com/img/add_to_slack.png)](https://slack.com/oauth/authorize?scope=commands,bot&client_id=12637824912.515475697794)
+
+Alternatively, you can install BotKube Slack app from [Slack app directory](https://slack.com/apps/AF5DZLHPC-botkube).
+
+### Add BotKube user to a Slack channel
+After installing BotKube app to your Slack workspace, you could see a new bot user with the name “BotKube” added in your workspace. Add that bot to a Slack channel you want to receive notification in.
+(You can add it by inviting @BotKube in a channel)
+
+
+
```

Comment:
Hey @3scava1i3r ,
We already mention to refer botkube.io for installation steps

Reply:
We don't need this change. Closing the PR

Reply:
i understand the installation guide part but the badges work right?